torchtext/datasets/text_classification.py

```diff
@@ -119,9 +120,9 @@ def _setup_datasets(dataset_name, root='.data', ngrams=1, vocab=None, include_un
 
     for fname in extracted_files:
         if fname.endswith('train.csv'):
-            train_csv_path = fname
+            train_csv_path = os.path.join(root, fname)
```

Comment:
This is BC breaking

Reply:
It is. See comment above about .zip and .tar files being handled by extract_archive

Reply:
Resolved by #692